src/pages/cart/components/product-list/ProductList.jsx

```diff
@@ -0,0 +1,66 @@
+import { useDispatch, useSelector } from "react-redux";
+import {
+  uncheckAllCheckButton,
+  checkAllCheckButton,
+  removeCheckedCartItem,
+} from "@/redux/modules/cartList";
+
+import StyledProductList from "@/pages/cart/components/product-list/ProductList.styled";
+import ProductItem from "@/pages/cart/components/product-item/ProductItem";
+import Checkbox from "@/components/checkbox/Checkbox";
+
+function ProductList() {
+  const cartList = useSelector((state) => state.cartListState);
+  const dispatch = useDispatch();
+
+  const allCheckboxIsChecked = () => {
+    const isAllChecked = cartList.every((item) => item.checked === true);
+    return isAllChecked;
+  };
+
+  const handleChange = (e) => {
+    const { checked } = e.target;
+
+    if (checked) {
+      dispatch(checkAllCheckButton());
+      return;
+    }
+    dispatch(uncheckAllCheckButton());
+  };
+
+  const handleClick = () => {
+    const isNonExistCheckedItem = cartList.every(
+      (item) => item.checked === false
+    );
+    if (isNonExistCheckedItem) return;
+
+    dispatch(removeCheckedCartItem());
+  };
+
+  return (
+    <StyledProductList>
+      <div>
+        <div className="checkbox-container">
+          <Checkbox
+            id="checkbox"
+            onChange={handleChange}
+            checked={allCheckboxIsChecked() ? true : false}
```

Comment:
```suggestion
            checked={allCheckboxIsChecked()}
```
allCheckboxIsChecked()에서 every 함수 결과가 boolean으로 리턴되기 때문에 삼항연산자로 true : false를 전달할 필요는 없어보이네요 😀

Reply:
생각해보니 true or false로 리턴되어서 삼항연산자를 사용할 필요는 없겠네요!